src/CMakeLists.txt

```diff
@@ -363,7 +363,7 @@ if(FORTE_C_INTERFACE)
     set_target_properties(${TARGET_NAME}
        PROPERTIES PUBLIC_HEADER ${CMAKE_CURRENT_SOURCE_DIR}/arch/c_interface/forte_c.h)
 
-    TARGET_LINK_LIBRARIES (${TARGET_NAME} PRIVATE ${FORTE_BASE_TARGET_NAME})
+    TARGET_LINK_LIBRARIES (${TARGET_NAME} PUBLIC ${FORTE_BASE_TARGET_NAME})
```

Comment:
I assume this is needed for the new C-Interface of FORTE to be linked correctly?

Reply:
exactly. The tests linking to these library weren't getting the right flags, in this case was the `-fsanitze=address` I think